src/main/java/Glendon/Glendon.java

```diff
@@ -0,0 +1,169 @@
+                response = in.nextLine();
+                break;
+            default:
+                try {
+                    checkResponse(response);
+                } catch (GlendonException e) {
+                    System.out.println("    Ayo no such commands.");
+                }
+                response = in.nextLine();
+                break;
+            }
+        }
+    }
+
+    private static void checkResponse(String response) throws GlendonException {
+        ArrayList<String> taskName = new ArrayList<String>();
+        taskName.add("todo");
+        taskName.add("deadline");
+        taskName.add("event");
+        taskName.add("mark");
+        taskName.add("unmark");
+        if(!taskName.contains(response)) {
+            throw new GlendonException();
+        }
+    }
+
+    private static void addEvent(String response) {
+        try{
+            String[] answers = response.split("/");
+            list[taskCounter] = new Event(answers[0].substring(6), answers[1].substring(5), answers[2].substring(3));
+            printAddedTask();
+            taskCounter++;
+        } catch (StringIndexOutOfBoundsException e) {
+            System.out.println("    Ayo you forgot to tell me what is the task");
+        }
+    }
+
+    private static void printAddedTask() {
+        System.out.println("    Got it. I've added this task:");
+        System.out.println("        " + list[taskCounter]);
+        System.out.println("    Now you have " + (taskCounter+1) + " tasks in the list.");
+    }
+
+    private static void addDeadline(String response) {
+        try {
+            String[] answers = response.split("/");
+            list[taskCounter] = new Deadline(answers[0].substring(9), answers[1].substring(3));
+            printAddedTask();
+            taskCounter++;
+        } catch (StringIndexOutOfBoundsException e) {
+            System.out.println("    Ayo you forgot to tell me what is the task");
+        }
+    }
+
+    private static void addTodo(String response) {
+        try {
+            list[taskCounter] = new Todo(response.substring(5));
+            printAddedTask();
+            taskCounter++;
+        } catch (StringIndexOutOfBoundsException e) {
+            System.out.println("    Ayo you forgot to tell me what is the task");
+        }
+    }
+
+    private static void unmarkDone(String response) {
+        try {
+            int taskValue = Integer.parseInt(response.split(" ")[1]) - 1;
+            list[taskValue].setCompleted(false);
+            System.out.println("    OK, I've marked this task as not done yet:");
+            System.out.println("        " + list[taskValue]);
+        } catch (ArrayIndexOutOfBoundsException e) {
+            System.out.println("    I can't read your mind, you need to tell me what your task number is.");
+        }
+    }
+
+    /**
+     * mark task as done
+     * @param response
+     */
+
+    private static void markDone(String response) {
```

Comment:
Maybe using markTask and unmarkTask is better here as unmarkDone is a bit confusing as unmarking usually means the task is not done.